scripts/install.sh

```
@@ -106,16 +108,33 @@ curl -fsSL https://deb.nodesource.com/setup_${V_NODE_VERSION} | sudo -E bash -
+fi
+
+if [[ -z "${V_PHP_VERSION}" ]]; then
+  echo "Error: could not determine a PHP version to install for '${V_DISTRO_CODENAME}'." && exit 1
+fi
+
+echo "Installing PHP ${V_PHP_VERSION}..."
+apt install -y php${V_PHP_VERSION} php${V_PHP_VERSION}-fpm php${V_PHP_VERSION}-mbstring php${V_PHP_VERSION}-gd php${V_PHP_VERSION}-xml php${V_PHP_VERSION}-curl php${V_PHP_VERSION}-zip php${V_PHP_VERSION}-bcmath php${V_PHP_VERSION}-soap php${V_PHP_VERSION}-redis php${V_PHP_VERSION}-sqlite3 php${V_PHP_VERSION}-intl
 if ! sed -i "s/www-data/vito/g" /etc/php/${V_PHP_VERSION}/fpm/pool.d/www.conf; then
   echo 'Error installing PHP' && exit 1
 fi
 service php${V_PHP_VERSION}-fpm enable
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)